client/my-sites/marketplace/pages/marketplace-product-install/index.tsx

```diff
@@ -56,11 +56,6 @@ import './style.scss';
 import { MarketplacePluginInstallProps } from './types';
 import type { IAppState } from 'calypso/state/types';
 
-interface InstalledPlugin {
```

Comment:
This type is no longer necessary because the type can now be correctly inferred.